examples/list-buckets.js

```diff
@@ -25,7 +25,12 @@ var s3Client = new Minio.Client({
   secretKey: 'YOUR-SECRETACCESSKEY',
 })
 
-s3Client.listBuckets(function (e, buckets) {
-  if (e) return console.log(e)
-  console.log('buckets :', buckets)
-})
+const listBucketsExample = async () => {
```

Comment:
can we just rename this file to mjs?

Reply:
also add a global notice about toplevel await， so code snippet can be more focused

Reply:
Thank you. I have updated.